student.rb

```diff
@@ -0,0 +1,15 @@
+require './person'
+require './classroom'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(age, name = 'Unknown', parent_permission = 'unknown')
+    super(age, name, parent_permission)
+    @classroom = classroom
+  end
```

Comment:
- [ ] what I meant with the previous review is that there needs to be a method which looks like the below example:
```
def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end

```